newproject.kts

```diff
@@ -0,0 +1,23 @@
+object NewProject {
```

Comment:
should we move `newproject.kts` to a directory? Maybe we will have `test.kts` for verifying `newproject.kts` script later 😄

Reply:
I suppose we could move it into `/scripts` or `/script`, what do you think? 🙏

Reply:
@Tuubz any of them is okay 🚀